Doc/Makefile

```
@@ -213,8 +213,9 @@ dist:
 	rm dist/python-$(DISTVERSION)-docs-texinfo.tar
 
 check:
-	$(SPHINXLINT) -i tools -i $(VENVDIR) -i README.rst
-	$(SPHINXLINT) ../Misc/NEWS.d/next/
+	# --severity=0 enables all checks, but we don't want to check for long lines
+	$(SPHINXLINT) -i tools -i $(VENVDIR) --severity=0 --disable='line too long'
```

```suggestion
	$(SPHINXLINT) -i tools -i $(VENVDIR) --enable default-role
```

Finally :)

I appreciate that the PR titled "Restore default role check" contains `--enable default-role` instead of `--disable line too long` :)